ConsoleGameLib/PhysicsTypes/PhysicsExtensions.cs

```diff
@@ -0,0 +1,59 @@
+﻿using ConsoleGameLib.CoreTypes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ConsoleGameLib.PhysicsTypes
+{
+    public static class PhysicsExtensions
+    {
+        public static bool ContainsPoint(this List<PhysicsPoint> points, Point point, bool mustInteractWithEnvironment = true)
```

Comment:
This looks like it could take an `IEnumerable<PhysicsPoint>` instead of a list, is there any particular reason you need it to be ordered?